Przypisanie do przykładów piwa

piotrowskislawomir opened the pull request.

Commit:
Very first dependency injection implementation

Commit:
introduced beer model and it's ingredients

piotrowskislawomir committed:
removed a lot of code and added algorithm A*

piotrowskislawomir committed:
changed algorithm A* to based on PriorityQueue

Commit:
Create README.md

piotrowskislawomir committed:
A* changed

piotrowskislawomir committed:
Przykłady piw przydział

piotrowskislawomir committed:
Przykłady Piw

Engilish Inida Pale Ale
American Amber Ale
Blonde Ale

mmanski commented on src/main/java/org/dszi/forklift/logic/GameLogic.java:
```diff
@@ -0,0 +1,101 @@
+        
+	private static final ArrayList<Task> _tasks = new ArrayList();
+
+        public GameLogic()
```
this is redundant

mmanski commented on src/main/java/org/dszi/forklift/logic/TreeState.java:
```diff
@@ -0,0 +1,130 @@
+    return srcPoint.x == descPoint.x && srcPoint.y == descPoint.y;
+}
+
+private int d(TreeItem item)
```
d? :)

mmanski commented on src/main/java/org/dszi/forklift/logic/TreeState.java:
```diff
@@ -0,0 +1,130 @@
+ */
+public class TreeState {
+
+public ArrayList<MoveActionTypes> treesearch(TreeItem startItem, Point destPoint, Grid grid)
```
it is better to return interface (java.util.List) instead of concrete impl

mmanski commented on src/main/java/org/dszi/forklift/logic/TreeState.java:
```diff
@@ -0,0 +1,130 @@
+
+public ArrayList<MoveActionTypes> treesearch(TreeItem startItem, Point destPoint, Grid grid)
+{
+    ArrayList<Point> points = new  ArrayList<>();
```
List<Point> points = new  ArrayList<>();

mmanski commented on src/main/java/org/dszi/forklift/logic/TreeState.java:
```diff
@@ -0,0 +1,130 @@
+        }
+    };
+
+private ArrayList<TreeItem> successor(Grid grid, TreeItem item)
```
List

concquerror committed:
Create Przykłady piw part2

Wybrałem środkowe piwa z listy tj. Dunkelweizen, Ameriacn Barleywine, German Pilsner

piotrowskislawomir committed:
cr fixes

Commit:
Saving results

Commit:
Merge

Commit:
Saving results

Commit:
Saving results

Commit:
Piwa

Commit:
All beers in one json

Commit:
fixed json

Commit:
Introduced decision tree for recognizing beer species
